test/bench/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
We can delete this now, since we're running `tsc` with no args its using the root config.

Reply:
We run currently run tsc using this config. We'd have to add the 'node' types into the root config if we want to be able to have it use console and other node APIs

Reply:
good call, okay sounds good to leave as is